sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Added feature to enable/disable SSL when initially creating connection to support amqp calls on port 5672.
```

Comment:
nit; nit; wondering if using _initially_ make any difference?

_Added feature to enable or disable SSL when creating connection to support AMQP calls on port 5672_

Reply:
Yes. From the [protocol guide](https://learn.microsoft.com/azure/service-bus-messaging/service-bus-amqp-protocol-guide), when you make your initial connection via port 5672, Service Bus will send a request to mandatory upgrade to SSL.  In the case of the emulator though, it won't force you to upgrade when you connect. I hope that helps. :)

Reply:
could you share an example of the connection string here? 
E.g. does it include endpoint (host and port)? also, I assume it does not have to be local - I can run emulator in a docker container and the hostname will be foobar, is this possible?

Reply:
I see, thanks for the clarification.